### DISCUSSION TITLE
Problems in installing aarch64 architecture

### DISCUSSION THREAD

**Discussion by puerteee** [OpenRadioss/OpenRadioss#925] [Q&A] [answered]
When compiling the starter, I encountered the problem shown in the figure. I hope to get help. Thank you.
![image](https://user-images.githubusercontent.com/121083779/208607715-01631e53-f244-49d2-ac58-1b8b9ce62c83.png)

**Comment by laurent-altr** [comment]
Hi, 
I have two guesses:
- Maybe the library has been corrupted during the download. Can you check the size of libmestis_linux64_gcc.a ? (Do you have git LFS installed ?). 
- Maybe the library we provide is not compatible with your environment. In that case you build your own library. You can download it [here](http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz) .

**Reply by puerteee** [reply]
Thanks,I have solved this problem.

**Comment by OlivierWienholtz** [comment, marked as answer]
Hi Puerteee

OpenRadioss is available on Arm (aarch64/) with Armflang & Armclang.
The build you have tried is for Gfortran / X86-64 

I recommend to install ArmFlang & ArmClang which are freely available & build with -arch=linuxa64.

Regards,
Olivier

**Reply by puerteee** [reply]
You are right,by this way I solved the problem,thank you!

**Reply by altairkubo** [reply]
Hello Olivier-san,

There appears to be a typo in the help message for specifying -arch when building the engine.

[opc@oci-pbscloud engine]$ ./build_script.sh -help
~
~

         -arch=linuxa64_gf               (SMP executable / Armflang compiler / Linux ARM64)
         -arch=linuxa64_gf -mpi=ompi     (OpenMPI executable / Armflang compiler / Linux X86-64)

~
~

I believe the correct answer is as follows.

         -arch=linuxa64               (SMP executable / Armflang compiler / Linux ARM64)
         -arch=linuxa64 -mpi=ompi     (OpenMPI executable / Armflang compiler / Linux X86-64)


Thanks,
Hiroji Kubo

**Reply by OlivierWienholtz** [reply]
Hello Hiroji-san,

This is correct. I'll fix it in the next days.
Thank You for pointing me out.

Best Regards,
Olivier W.
